advanced/Scripts/utils.sh

```diff
@@ -86,13 +86,17 @@ getFTLConfigValue(){
 # setFTLConfigValue dns.upstreams '[ "8.8.8.8" , "8.8.4.4" ]'
 #######################
 setFTLConfigValue(){
-  pihole-FTL --config "${1}" "${2}" >/dev/null
-  local ret=$?
-  if [ ${ret} = 5 ]; then
-    printf "  %s %s set by environment variable. Please unset it to use this function\n" "${CROSS}" "${1}"
-    exit 5
-  elif [ ${ret} != 0 ]; then
-    printf "  %s Failed to set %s. Try with sudo power\n" "${CROSS}" "${1}"
-    exit 1
-  fi
+    local err
+    { pihole-FTL --config "${1}" "${2}" >/dev/null; err="$?"; } || true
```

Comment:
You can declare that `local` variable in the grouping if you'd like.
```suggestion
    { pihole-FTL --config "${1}" "${2}" >/dev/null; local err="$?"; } || true
```